pkg/sync/filepath_sync_test.go

```diff
@@ -117,8 +117,9 @@ func TestFilePathSync_Fetch(t *testing.T) {
 	}{
 		"success": {
 			fpSync: sync.FilePathSync{
-				URI:    fmt.Sprintf("%s/%s", dirName, fetchFileName),
-				Logger: logger.NewLogger(nil, false),
+				URI:      fmt.Sprintf("%s/%s", dirName, fetchFileName),
+				Logger:   logger.NewLogger(nil, false),
+				FileType: "json", // this is the default
```

Comment:
We aren't using `FilePathSync`'s `Fetch` to fetch files via non-command line means (in which case `FileType` won't be set to `json` by default) as of now AFAIK. Might change in the future. If someone has a better approach, I would like know.